apollo-rx2support/src/main/java/com/apollographql/apollo/rx2/Rx2Apollo.java

```diff
@@ -66,30 +63,39 @@ private Rx2Apollo() {
   }
 
   /**
-   * Converts an {@link ApolloCall} to a synchronous Single.
+   * Converts an {@link ApolloCall} to an {@link Observable}.
```

Comment:
I guess we should provide more info why it's `Observable` on cache and network fetch strategy example